gst/nnstreamer/include/nnstreamer_plugin_api.h

```diff
@@ -514,13 +514,33 @@ gst_tensor_meta_info_append_header (GstTensorMetaInfo * meta, GstMemory * mem);
 
 /**
  * @brief Convert GstTensorMetaInfo structure to GstTensorInfo.
- * @param meta[in] tensor meta structure to be converted
- * @param info[out] GstTensorInfo to be filled
+ * @param[in] meta tensor meta structure to be converted
+ * @param[out] info GstTensorInfo to be filled
  * @return TRUE if successfully set the info
  */
 gboolean
 gst_tensor_meta_info_convert (GstTensorMetaInfo * meta, GstTensorInfo * info);
 
+/**
+ * @brief NYI. Make dense tensor with input sparse tensor.
+ * @param[in,out] meta tensor meta structure to be updated
+ * @param[in] in pointer of input sparse tensor data
+ * @param[out] out pointer of output dense tensor data
+ * @return TRUE if no error
+ */
+extern gboolean
+gst_tensor_sparse_to_dense (GstTensorMetaInfo * meta, gpointer in, gpointer out);
```

Comment:
As we talked on offline, move these functions to internal.

Reply:
Added a internal header tensor_sparse_util.h for util functions that will be used in related elements. PTAL